__tests__/commands/info.js

```diff
@@ -42,16 +42,18 @@ const expectedKeys = [
   'bugs',
   'license',
   'scripts',
-  'dependencies',
-  'devDependencies',
   'dist',
   'directories',
 ];
 
+// yarn now ships as built, single JS files so it has no dependencies
```

Comment:
I'm still confused as to why this wasn't broken in my original PR? The builds on #3866 are clean :/